src/meta/ps2_npsf.c

```diff
@@ -14,7 +15,9 @@ VGMSTREAM * init_vgmstream_ps2_npsf(STREAMFILE *streamFile) {
 
     /* check extension, case insensitive */
     streamFile->get_name(streamFile,filename,sizeof(filename));
-    if (strcasecmp("npsf",filename_extension(filename))) goto fail;
+    if (strcasecmp("npsf",filename_extension(filename)) &&
```

Comment:
Well. Since I'm sort of self-evaluating the viability of header.c this one was a quick update.

I'm not super-confident adding a new "main" src file into a venerable structure, which in the end only I will use. I'm considering just moving its funcs into streamfile.c/util.c/vgmstream.c for simplicity.

formats.c I'm quite satisfied tho, couldn't accept all those gigantic if/switch-case lists.